Dockerfile

```diff
@@ -95,8 +98,6 @@ RUN apt-get update && \
     tini=0.19.* \
     build-essential \
     git=1:2.* \
-    curl \
-    ca-certificates \
```

Comment:
i dont think we need these, and they were added in https://github.com/PrefectHQ/prefect/pull/16235, not before